plugins/search/src/components/SidebarSearch/SidebarSearch.tsx

```diff
@@ -19,9 +19,13 @@ import { useNavigate } from 'react-router-dom';
 import { rootRouteRef } from '../../plugin';
 
 import { SidebarSearchField } from '@backstage/core-components';
-import { useRouteRef } from '@backstage/core-plugin-api';
+import { useRouteRef, IconComponent } from '@backstage/core-plugin-api';
 
-export const SidebarSearch = () => {
+export type SidebarSearchProps = {
```

Comment:
And this type too needs exporting the whole way out of the package too.

Reply:
exported, please check again.